DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: scRNAseq
 Title: Collection of Public Single-Cell RNA-Seq Datasets
-Version: 2.1.6
-Date: 2020-02-14
+Version: 2.2.0
```

Comment:
Don't do that, this is bumped during BioC release.

Reply:
Date as well?

Reply:
The new date is fine.